tests/translator/output/aws-cn/http_api_openapi_with_propagate_tags.json

```diff
@@ -0,0 +1,112 @@
+{
+  "AWSTemplateFormatVersion": "2010-09-09",
+  "Resources": {
+    "MyHttpApi": {
```

Comment:
For my own understanding, before the `Tags` property for this resource was being set which caused the deployment error because `x-amazon-apigateway-tag-value` is also set in the Body?

Reply:
Yep exactly. We already add the tags to the OpenApi definition (`Body` property) [here](https://github.com/gracelu0/serverless-application-model/blob/develop/samtranslator/model/api/http_api_generator.py#L522) when constructing the resource. We were additionally adding the `Tags` property to `AWS::ApiGatewayV2::Api` when `PropagateTags` is set to `True` which is what caused the error.